packages/components/src/TagGroup/TagGroup.tsx

```diff
@@ -43,12 +51,15 @@ const _TagGroup = ({
       <TagList
         items={items}
         className={classNames.listItems}
-        renderEmptyState={renderEmptyState}
+        renderEmptyState={emptyState}
       >
         {children}
       </TagList>
+      {name ? <TagGroupHiddenInput name={name} /> : null}
     </FieldBase>
   );
 };
 
 export { _TagGroup as TagGroup };
+
+// TODO: Why is the helptext showing?
```

Comment:
because its a form component, and they have a help text? Not sure what you mean here 🙈

Reply:
ah sorry, this was from the story. There is a bottom/top margin because of the help text. Will go an check.